### PR TITLE
Add Umami

### PR DESCRIPTION
**Pull request by sharknoon**
# App Submission

### App name
Umami

### 256x256 SVG icon
umami

### Gallery images
Screenshots are from the [official demo](https://analytics.umami.is/share/LGazGOecbDtaIwDr/umami.is)

### I have tested my app on:
- [ ] umbrelOS on a Raspberry Pi
- [ ] umbrelOS on an Umbrel Home
- [x] umbrelOS on Linux VM

### CODE REVIEW

File: umami/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: umami_app_1
+      APP_PORT: 3000
+      PROXY_AUTH_ADD: "false"
+  
+  app:
```
Comment (nmfretz) [inline umami/docker-compose.yml:10]: Looking at the[ Dockerfile for Umami](https://github.com/umami-software/umami/blob/7a75639dd3d7aeff46104b71ebfb3853fc0eee09/Dockerfile#L54), we can easily override the default behaviour of this container to run as the `umbrel` user 1000 by adding this line:

`user: "1000:1000"`

**Review by nmfretz — COMMENTED** [review]
Thanks for this submission @sharknoon! Super high quality PR, really great work. 

I've left some minor suggestions in the review below. We'll start getting the gallery images ready so this can go live on the app store.

One thing for us to consider is how to communicate to users that they may need to expose umami to the web in order to use the app properly with websites they've deployed. This could be as simple as just adding a short note in the app description.

Domain names and https are on the horizon for umbrelOS, but right now the easiest way for users to do this would be to install the [cloudflare app](https://apps.umbrel.com/app/cloudflared) from the umbrel app store, and expose the umami app using their own domain.

File: umami/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      PROXY_AUTH_ADD: "false"
+  
+  app:
+    image: ghcr.io/umami-software/umami:postgresql-latest@sha256:4fd4bbf741a10c1109eb8387a8aa90f9a40d673972dc268ebd97e41e5b045c0c
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      DATABASE_URL: postgresql://umami:umami@db:5432/umami
+      DATABASE_TYPE: postgresql
+      APP_SECRET: ${APP_SEED}
```
[the remark this answers is not in the document]
Comment (sharknoon) [reply umami/docker-compose.yml:17]: thank you 😃

File: umami/umbrel-app.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+manifestVersion: 1
+id: umami
+name: Umami
+tagline: Umami is a simple, fast, privacy-focused alternative to Google Analytics
+category: developer
+version: "2.11.3"
+port: 25727
+description: >-
```
[the remark this answers is not in the document]
Comment (sharknoon) [reply umami/umbrel-app.yml:8]: thank you so much 😃

File: umami/umbrel-app.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+submission: https://github.com/getumbrel/umbrel-apps/pull/1083
+repo: https://github.com/umami-software/umami
+support: https://github.com/umami-software/umami/discussions
+gallery:
+  - 1.png
+  - 2.png
+  - 3.png
+  - 4.png
+defaultUsername: admin
+defaultPassword: umami
```
[the remark this answers is not in the document]
Comment (sharknoon) [reply umami/umbrel-app.yml:45]: Thank you for the hint. I knew about the feature, but unfortunately there seems to be no [environment variable](https://umami.is/docs/environment-variables) that let's me configure a default user.

**Comment by sharknoon** [comment]
Thank you @nmfretz for your review. I have implemented the necessary suggestions 😃

**Comment by nmfretz** [comment]
> Thank you @nmfretz for your review. I have implemented the necessary suggestions 😃

Wonderful. Thanks @sharknoon. Gallery assets have been created and are stored here: https://github.com/getumbrel/umbrel-apps-gallery/tree/master/umami

I have made some minor changes to finalize things and we are good to launch.

Thanks again for your work on Umami!

**Comment by sharknoon** [comment]
It was my pleasure 👍

**Pull request merged**